src/sql/workbench/parts/grid/views/editData/editData.component.ts

```diff
@@ -177,9 +177,9 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 		this.overrideCellFn = (rowNumber, columnId, value?, data?): string => {
 			let returnVal = '';
 			if (Services.DBCellValue.isDBCellValue(value)) {
-				returnVal = value.displayValue;
+				returnVal = value.displayValue.replace(/(\r\n|\n|\r)/g, ' ');
```

Comment:
we should introduce a method to handle the line break to space conversion logic, and please add comments explaining why this is added

Reply:
I'd update #1303 with this information too and then we should definitely have that as a fix we take for June.